src/main/proto/inter_node_rpc_service.proto

```diff
@@ -69,6 +69,11 @@ enum ResourceEnum {
   // threadpool
   WRITE_THREADPOOL = 4 [(additional_fields).name = "write threadpool"];
   SEARCH_THREADPOOL = 5 [(additional_fields).name = "search threadpool"];
+
+  // Cache
+  FIELD_DATA_CACHE = 6 [(additional_fields).name = "field data cache"];
```

Comment:
let's reserve some field numbers for thread pool and many be we can start from 10 here ?